peerme/euroix_json.py

```diff
@@ -47,7 +47,7 @@ def __init__(self, config, refresh_data=False, loop=None):
 
         return url, data
 
-    def fetch_json(self, ixp_json_file, timeout=10):
+    def fetch_json(self, ixp_json_file, timeout=30):
```

Comment:
^^ Use the passed in timeout in this function

Another diff can actually pass it in from config ...